canary/ClientApp/src/components/dashboard/Dashboard.js

```diff
@@ -72,6 +72,12 @@ export class Dashboard extends Component {
                   description="Test a data provider system's ability to produce pre-defined records as tested at Connectathons."
                   route="test-connectathon-dash/records"
                 />
+                 <DashboardItem
+                  icon="tasks"
+                  title="Validate FHIR Death Records (Producing)"

```

Comment:
Should this also be called `Syntax Checker` and follow the rules we laid out today (FHIR Death Records -> FHIR VRDR Records)?

Also the fact that this checks against an IJE seems important enough to call out in the title right?

Reply:
In this case, it is checking whether the values are correct so I think it is more of a validator. Doesn't the syntax check just check for correct formatting? I will add in IJE to the title and change it to VRDR Records